easylinks/src/main/java/com/google/step/servlets/ManageServlet.java

```diff
@@ -0,0 +1,79 @@
+package com.google.step.servlets;
+
+import com.google.step.servlets.ServletHelper;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Define an easylink object. */
+final class EasyLink {
+    
+  public EasyLink(long id, String shortcut, String url, String email) {
+    this.id = id;
+    this.shortcut = shortcut;
+    this.url = url;
+    this.email = email;
+  }
+
+  private final long id;
+  private final String shortcut;
+  private final String url;
+  private final String email;
+}
```

Comment:
Probably we don't need this inner class. We could convert the whole `Entity` by using `gson.convertToJson(Entity)`. Then, in the javascript , we can access the property of the entity by using `entity. propertyMap `. For example, say we want access the user email, we could write `var email = entity. propertyMap.email.`

Reply:
done!